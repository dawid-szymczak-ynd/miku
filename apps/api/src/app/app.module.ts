import { ApiCommonModule } from '@miku-credit/api/common';
import { Module } from '@nestjs/common';
import { TerminusModule } from '@nestjs/terminus';

@Module({
  imports: [TerminusModule, ApiCommonModule],
  controllers: [],
  providers: [],
})
export class AppModule {}
