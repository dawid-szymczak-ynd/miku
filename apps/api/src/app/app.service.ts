import { Message } from '@miku-credit/api-interfaces';
import { Injectable } from '@nestjs/common';

@Injectable()
export class AppService {
  public getData(): Message {
    return { message: 'Welcome to api!!!' };
  }
}
