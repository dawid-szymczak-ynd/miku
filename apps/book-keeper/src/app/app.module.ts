import { BookKeeperCommonModule } from '@miku-credit/book-keeper/common';
import { Module } from '@nestjs/common';
import { TerminusModule } from '@nestjs/terminus';
import { TypeOrmModule } from '@nestjs/typeorm';

@Module({
  imports: [
    TypeOrmModule.forRoot({
      type: 'postgres',
      host: process.env.POSTGRESQL_HOST,
      port: Number(process.env.POSTGRESQL_PORT),
      username: process.env.POSTGRESQL_USERNAME,
      password: process.env.POSTGRESQL_PASSWORD,
      database: process.env.POSTGRESQL_DATABASE,
      autoLoadEntities: true,
    }),
    TerminusModule,
    BookKeeperCommonModule,
  ],
  controllers: [],
  providers: [],
})
export class AppModule {}
