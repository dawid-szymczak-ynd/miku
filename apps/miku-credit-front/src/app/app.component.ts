import { Component } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { of } from 'rxjs';

@Component({
  selector: 'miku-credit-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.scss'],
})
export class AppComponent {
  hello$ = of('MockMessage!');
  constructor(private http: HttpClient) {}
}
