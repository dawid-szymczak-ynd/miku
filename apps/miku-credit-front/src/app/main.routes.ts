import { Route } from '@angular/router';

export const routes: Route[] = [
  {
    path: '',
    loadChildren: () =>
      import('@miku-credit/miku-credit-front/homepage').then(
        (module) => module.MikuCreditFrontHomepageModule
      ),
  },
];
