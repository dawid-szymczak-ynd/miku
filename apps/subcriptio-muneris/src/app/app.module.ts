import { SubcriptioMunerisCommonModule } from '@miku-credit/subcriptio-muneris/common';
import { Module } from '@nestjs/common';
import { TypeOrmModule } from '@nestjs/typeorm';

@Module({
  imports: [
    TypeOrmModule.forRoot({
      type: 'postgres',
      host: process.env.POSTGRESQL_HOST,
      port: Number(process.env.POSTGRESQL_PORT),
      username: process.env.POSTGRESQL_USERNAME,
      password: process.env.POSTGRESQL_PASSWORD,
      database: process.env.POSTGRESQL_DATABASE,
      autoLoadEntities: true,
    }),
    SubcriptioMunerisCommonModule,
  ],
  controllers: [],
  providers: [],
})
export class AppModule {}
