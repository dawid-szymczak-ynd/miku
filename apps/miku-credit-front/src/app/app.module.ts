import { HttpClientModule } from '@angular/common/http';
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
import { RouterModule } from '@angular/router';
import { MikuCreditFrontLoanModule } from '@miku-credit/miku-credit-front/loan';
import { MikuCreditFrontPaybackModule } from '@miku-credit/miku-credit-front/payback';
import { EffectsModule } from '@ngrx/effects';
import { routerReducer, StoreRouterConnectingModule } from '@ngrx/router-store';
import { StoreModule } from '@ngrx/store';
import { StoreDevtoolsModule } from '@ngrx/store-devtools';
import { NxModule } from '@nrwl/angular';

import { environment } from '../environments/environment';
import { AppComponent } from './app.component';
import { routes } from './main.routes';

@NgModule({
  declarations: [AppComponent],
  imports: [
    BrowserModule,
    HttpClientModule,
    RouterModule.forRoot(routes),
    BrowserAnimationsModule,
    NxModule.forRoot(),
    StoreModule.forRoot({ router: routerReducer }),
    EffectsModule.forRoot([]),
    StoreRouterConnectingModule.forRoot(),
    StoreDevtoolsModule.instrument({
      name: 'Registration state',
      logOnly: environment.production,
    }),
    HttpClientModule,
    MikuCreditFrontLoanModule,
    MikuCreditFrontPaybackModule,
  ],
  providers: [],
  bootstrap: [AppComponent],
})
export class AppModule {}
