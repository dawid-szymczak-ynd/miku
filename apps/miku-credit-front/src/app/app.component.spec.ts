import { BrowserAnimationsModule, NoopAnimationsModule } from '@angular/platform-browser/animations';
import { RouterModule } from '@angular/router';
import { RouterTestingModule } from '@angular/router/testing';

import { Shallow } from 'shallow-render';

import { AppComponent } from './app.component';
import { AppModule } from './app.module';

describe('AppComponent', () => {
  let shallow: Shallow<AppComponent>;

  beforeEach(() => {
    shallow = new Shallow(AppComponent, AppModule);
  });

  it('should match snapshot', async () => {
    const { fixture } = await shallow
      .replaceModule(
        RouterModule,
        RouterTestingModule.withRoutes([
          {
            path: '',
            redirectTo: '/test',
            pathMatch: 'full',
          },
        ])
      )
      .replaceModule(BrowserAnimationsModule, NoopAnimationsModule)
      .render();

    expect(fixture).toMatchSnapshot();
  });
});
