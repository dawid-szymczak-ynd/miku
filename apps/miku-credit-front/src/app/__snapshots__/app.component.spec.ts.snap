// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should match snapshot 1`] = `
<ng-component>
  <miku-credit-root>
    <router-outlet />
  </miku-credit-root>
</ng-component>
`;
